addons/ofxOsc/src/ofxOscMessage.h

```diff
@@ -9,13 +9,11 @@
 class ofxOscMessage {
 public:
 	ofxOscMessage();
-	ofxOscMessage(std::string address)
-		: ofxOscMessage() {
-		setAddress(address);
-	}
 	~ofxOscMessage();
 	ofxOscMessage(const ofxOscMessage & other);
-	ofxOscMessage(std::string & address);
+	ofxOscMessage(const std::string address) { setAddress(address); }
```

Comment:
isn't `const std::string &address` ...?

Reply:
well I've was struggling with the ambiguous constructor problem stemming in another (non-core) templated class, but I think removing the non-& is what actually fixes the case, which means const can be back (but is it OK to have no non-& string? I guess for OSC paths it's OK? or maybe there are cases I don't foresee)

Reply:
NB working in C++20 since a few weeks maybe some construction/type interpretations are different. (mostly using ranges and format. note that std::format requires macOS13.3, which I don't mind but is probably a no-go for OF adoption (rule of 3 versions would mean minimally support macOS11— if we want format before 2027 probably better to include the original {fmt} (maybe alias it as ofFormat like ofJson to allow eventual switch)).

Reply:
if argument is non const reference (`const std::string`) then doesn't those make ambiguous?

sample code:

```cpp
#include <string>
#include <iostream>

void f(const std::string str) {
	std::cout << "const string" << std::endl;
}

void f(std::string &&str) {
	std::cout << "move" << std::endl;
}

int main(int argc, char *argv[]) {
	f("hoge"); // Error: call to 'f' is ambiguous...
	f(std::string{"hoge"}); // Error: call to 'f' is ambiguous...
	std::string str{"hoge"};
	f(str); // OK (call with const std::string)
	f(std::move(str)); // Error: call to 'f' is ambiguous...
}
```